src/auth_db/test.rs

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+
+use super::*;
+
+#[test]
+fn get_email_bounces()
+{
+  let settings = Settings::new().expect("config error");
+  let db = DbClient::new(&settings);
+  if let Err(_error) = db.get_email_bounces("foo@example.com") {
+    assert!(false, "DbClient::get_email_bounces should not have failed");
+  }
+}
+
+#[test]
+fn get_email_bounces_invalid_address()
+{
+  let settings = Settings::new().expect("config error");
+  let db = DbClient::new(&settings);
+  match db.get_email_bounces("") {
+    Ok(_) => assert!(false, "DbClient::get_email_bounces should have failed"),
+    Err(error) => assert_eq!(error.description(), "auth db response: 400 Bad Request"),
+  }
+}
```

Comment:
There aren't many test cases here because we only have a method for reading. When we implement #5, we can improve the coverage by asserting that the result from `get_email_bounces` matches what we write with `create_email_bounce`.